x-pack/test/rule_registry/security_and_spaces/tests/trial/update_alert.ts

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import expect from '@kbn/expect';
+
+import {
+  superUser,
+  obsMinReadAlertsAll,
+  obsMinReadAlertsAllSpacesAll,
+  obsMinAll,
+  obsMinAllSpacesAll,
+  obsAlertsAll,
+  obsAlertsAllSpacesAll,
+} from '../../../common/lib/authentication/users';
+import type { User } from '../../../common/lib/authentication/types';
+import { FtrProviderContext } from '../../../common/ftr_provider_context';
+import { getSpaceUrlPrefix } from '../../../common/lib/authentication/spaces';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext) => {
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+  const esArchiver = getService('esArchiver');
+
+  const TEST_URL = '/internal/rac/alerts';
+  const ALERTS_INDEX_URL = `${TEST_URL}/index`;
+  const SPACE1 = 'space1';
```

Comment:
I don't see any integration test cases for `SPACE2` (in the "trial" suite here, or in the "basic" suite). It would be a good idea to add those as well.

Reply:
space2 tests here: https://github.com/elastic/kibana/pull/100705/files#diff-af7c4dd80893f1c26ff8f06abf5e952b89b648abfe64b7c2b2594c8acb51357bR44